admin/src/pages/View/components/NavigationItemForm/index.js

```diff
@@ -32,16 +37,29 @@ const NavigationItemForm = ({
   onCancel,
   getContentTypeEntities,
   usedContentTypesData,
-  appendLabelPublicationStatus = () => '',
+  appendLabelPublicationStatus = appendLabelPublicationStatusFallback, 
   locale,
+  readNavigationItemFromLocale,
 }) => {
+  const [isLoading, setIsLoading] = useState(isLoading);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(isPreloading);
```

_Uncaught ReferenceError: Cannot access 'isLoading' before initialization_

Reply:
Updated